src/pages/users/user.tsx

```
@@ -1,15 +1,92 @@
+    title: "Email",
+    dataIndex: "email",
+    key: "email",
+  },
+  {
+    title: "Role",
+    dataIndex: "role",
+    key: "role",
+  },
+];
 
 const User = () => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'User'. Consider to delete it or rename it.

'User' is defined here:


(lint/suspicious/noRedeclare)